code/style.css

```diff
@@ -1,4 +1,149 @@
-html {
-  background: #0025ff;
-  color: #fff;
+@import url('https://fonts.googleapis.com/css2?family=Architects+Daughter&family=Bebas+Neue&family=Shadows+Into+Light&display=swap');
+body {
+	margin: 0 0;
+	font-family: 'Bebas Neue', cursive;
 }
+.backgroundgif {
+	width: 100%;
+	height: 400px;
+}
+/*Stylig for the headersection using flexbox*/
+
+header {
+	margin: 5px;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+	font-size: 30px;
+}
+.logo {
+	height: 40px;
+	width: auto;
+}
+.navbar {
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+}
+.logoandtext {
+	display: flex;
+	align-items: center;
+	flex-grow: 1;
+}
+.links {
+	display: flex;
+}
+.logo-hidden {
+	display: none;
+}
+a {
+	text-decoration: none;
+	color: black;
+	padding: 10px;
+	font-size: 30px
+}
+a:hover {
+	background-color: rgb(130, 141, 151);
+}
+/* Stylig for the mainnews section using grid*/
+
+.main {
+	width: 80%;
+	margin: auto
+}
+h1 {
+	margin: 0 0;
+	font-size: 3.5vw;
+	color: whitesmoke;
+}
+.big-news {
+	display: grid;
+	grid-template-columns: 50% 50%;
+	grid-gap: 0px;
+	margin-top: 80px;
+	margin-bottom: 150px;
+	justify-items: center;
+	align-items: center;
+	background-color: rgb(130, 141, 151);
+}
+.bignewspics {
+	width: 100%;
+}
+.galleryimg {
+	width: 100%;
+	height: auto;
+	object-fit: contain;
+}
+.newsgrid {
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(250px, 1fr));
+	margin-bottom: 80px;
+	justify-items: center;
+}
+figcaption {
```

Comment:
Really nice to se your solution whit using figcaption and figure in html. I learned something new, thanks!